Groups: Wall: add suggestions

lalka2016 opened the pull request:
Данный пр добавляет предложку в группы. Её можно включить, задав "Стену" 
![и](https://github.com/openvk/openvk/assets/99399973/6e6f1050-b84f-4ced-ac3d-1143593cf948)
![з](https://github.com/openvk/openvk/assets/99399973/1248a536-1e59-4096-81bc-2f326b5776c9)
![о](https://github.com/openvk/openvk/assets/99399973/9713af93-f0b6-468c-9f7c-5018abfa4a7f)
Как и в вк, можно поменять текст поста и включить/выключить отображение автора
![б](https://github.com/openvk/openvk/assets/99399973/24608352-993f-4e5e-bf18-fbd16c7a7468)
После принятия поста автору придёт уведомление
![р](https://github.com/openvk/openvk/assets/99399973/eefb2084-844a-49ca-8a97-6302d6b81338)
![а](https://github.com/openvk/openvk/assets/99399973/97f9ca56-75ea-4a47-88c3-e033ff9960e1)
![ж](https://github.com/openvk/openvk/assets/99399973/18e84d0c-b13f-4b52-b2c6-fec215e2b3a2)
![е](https://github.com/openvk/openvk/assets/99399973/fa104dc0-5c08-45b8-9707-4e59afcc0def)
Если число предложенных постов делится без остатка на 10, то администраторам группы придёт уведомление
![н](https://github.com/openvk/openvk/assets/99399973/94503970-576a-48fe-a14c-8ed0d3565c27)
Так же число записей в предложке будет отображаться у закреплённых групп
![и](https://github.com/openvk/openvk/assets/99399973/98ef46a3-4239-4c83-8cfe-4c3be8f558aa)
Добавлены вкладки стены: записи владельца стены и чужие записи.
![е](https://github.com/openvk/openvk/assets/99399973/54ebc079-362f-4056-a08f-db2a270c42fb)
![изображение](https://github.com/openvk/openvk/assets/99399973/352f3d38-e5b3-4730-84b6-0d0c9a89d957)
Ещё теперь если у вас включён микроблог, то у подписанных постов вместо "Автор" будет иконка с человечком
![изображение](https://github.com/openvk/openvk/assets/99399973/a7575f89-3257-4c23-a3e8-85f931775c5d)

У wall.post добавлен параметр post_id для публикации админам группы предложенной записи, добавлен метод wall.delete, у wall.get добавлен параметр filter (работает всё кроме postponed и... donut), у groups.get и groups.getById добавлены поля can_suggest и suggest_count (нету в vkapi), в объект поста добавлен signer_id (если пост подписан), пофикшен метод groups.edit. Ну и ещё пофикшено прикрепление заметки в хроме (опять?)

Resolves #232 and resolves #704

Commit:
Wall: add early suggestions

Commit:
Fix br

Commit:
Fix empty posts

Commit:
fck

Commit:
Add offset for api

Commit:
Add notifications of new suggestion posts

Commit:
Fix mentions in suggested posts

Commit:
🤮🤢

Commit:
Change regex

Теперь оно удаляет все теги а не только <br>

Commit:
Add da koroche pohuy

Commit:
Эдд апи метходс

Методы нестандартные немного

Commit:
Pon

Commit:
Add skloneniyia

Commit:
newlines

Commit:
int

Commit:
Update loaders and add avtopodgruzka postov

Commit:
Update JOERGK.strings

Commit:
Blin

Commit:
Remove repeated code, fix loaded buttons on chr...

...ome and fix getting suggested posts via API.Wall.getPost

Commit:
Merge branch 'master' into suggests

n1rwana reviewed (changes requested):
Думаю, не должно быть возможности голосовать в опросе из неопубликованной предложенной записи

![image](https://github.com/openvk/openvk/assets/93197434/51ce44e5-cc4d-4931-aa70-f8940eb609cb)

Commit:
Fix polls

Commit:
Merge branch 'suggests' of https://github.com/lalka2016/openvkuh into suggests

Commit:
Fihes

Теперь уведомление о принятии поста не приходит, если вы приняли свой же пост

Пофикшен баг перехода в предложку

Добавлен старый вид постов в предложке

Теперь счётчик постов в предложке у прикреплённой группы обновляется при принятии или отклонении поста

Убрано всплывающее уведомление об отклонении поста (оно раздражает)

Теперь если вы посмотрели все посты на одной странице (не на первой) и на ней не осталось постов, вас телепортирует на предыдущую страницу

Commit:
Merge branch 'master' into suggests

n1rwana reviewed (dismissed):
Через API пользователь может удалить предложенную запись, **когда она уже опубликована на стене сообщества**

Commit:
Remove ability to delete your accepted psto

Commit:
oi blin

Commit:
Improvements 2 api

veselcraft reviewed (changes requested):
впринципе неплохо всё но есть пару вопрсов

Commit:
g

Commit:
openvk.uk

Возможно, приведение кода к кодстайлу (удаление скобочек то есть)

Commit:
aiaks

Commit:
al_wall.js -> al_suggestions.js

veselcraft reviewed (approved):
выглядит неплохо, посмотрим что @celestora скажет

Commit:
👨‍💻 Add 👨‍💻 fading 👨‍💻

Commit:
Add "owner's posts' and "other's posts"

Давайте рофлить👨‍💻👨‍💻👨‍💻

Commit:
planshet openvk

Add tabs for post view, add signer's object in wall get and add person icon in microblog

Commit:
Merge branch 'master' into suggests

Commit:
Simplefai ze kod

Commit:
Merge branch 'master' into suggests

Commit:
PHP 8 FIX WATAFAK

Commit:
Add indesk
